### PR TITLE
Registration test

### PR DESCRIPTION
**Pull request by azizullahsaeidi**
1. Create Registration Test
2. Create Model Factory
3. Create TaxonomyTermFactory
4. Create LanguageEnum
5. Create GenderEnum
6. Create TaxonomyVocabularyEnum

### CODE REVIEW

File: app/Enums/LanguageEnum.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Enums;
+
+enum LanguageEnum: string
+{
+    case Farsi = 'fa';
+    case sowji = 'sw';
```
Comment (uyscuti-wiki) [inline app/Enums/LanguageEnum.php:8]: Why is the 's' in 'sowji' lowercase?

**Review by uyscuti-wiki — APPROVED** [review]
Great work Aziz. This looks good. Please address the one comment before I merge this.

File: app/Enums/LanguageEnum.php (hunk 1 above)
Comment (azizullahsaeidi) [reply app/Enums/LanguageEnum.php]: Thank you @uyscuti-wiki , I missed to make it uppercase.

**Comment by azizullahsaeidi** [comment]
> Great work Aziz. This looks good. Please address the one comment before I merge this.

Thank you @uyscuti-wiki , Done

**Review by uyscuti-wiki — APPROVED** [review]

**Pull request merged**